### PR TITLE
Update to 2024.1

### PR DESCRIPTION
**Pull request by Bramfoxie**
- Fixed a bug (typo) in the color code.
- Update to 2024.1

@lankydan please check. 😊

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
 pluginName_ = palenight-intellij-theme
-pluginVersion = 1.0.2
+pluginVersion = 1.0.3
 pluginSinceBuild = 193
-pluginUntilBuild = 233.*
+pluginUntilBuild = 241.*
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions
-pluginVerifierIdeVersions = 2022.2, 2022.3, 2023.1.1, 2023.3
+pluginVerifierIdeVersions = 2022.2, 2022.3, 2023.1.1, 2023.3, 2024.1
```
Comment (lankydan) [inline gradle.properties]: Remove `2022.2` from this list, that _should_ allow the tests to run correctly.

File: src/main/resources/themes/palenight-theme.json [hunk 2]
```diff
@@ -201,7 +201,7 @@
       "hoverColor": "#444267",
       "hoverMaskColor": "#444267",
       "inactiveMaskColor": "#292D3E",
-      "inactiveColoredFileBackground": "#292D3E2",
+      "inactiveColoredFileBackground": "#292D3E",
```
Comment (lankydan) [inline src/main/resources/themes/palenight-theme.json]: What happened because of this change?

Comment (Bramfoxie) [reply src/main/resources/themes/palenight-theme.json]: There was a warning in the console that the color could not be parsed. I checked, this code does not match the color in HEX. I assumed it was a typo.

Comment (lankydan) [reply src/main/resources/themes/palenight-theme.json]: Odd, iirc that last number should control the opacity of the colour.

Comment (Bramfoxie) [reply src/main/resources/themes/palenight-theme.json]: #RRGGBBAA or #RGBA. Maybe it should be #292D3E02. I'll check now...

**Comment by Bramfoxie** [comment]
What's wrong again? 😢

File: src/main/resources/themes/palenight-theme.json (hunk 2 above)
Comment (lankydan) [reply src/main/resources/themes/palenight-theme.json]: You might be right, its been a while... But worth checking at least.

**Comment by lankydan** [comment]
> What's wrong again? 😢

I think I'll have to take a look at it sometime.

I _think_ its due to APIs being removed in 2024 versions that the plugin is currently using.

Updating some of the dependency versions might fix this but I'd need to have a look / trial and error to figure it out.

File: src/main/resources/themes/palenight-theme.json (hunk 2 above)
Comment (Bramfoxie) [reply src/main/resources/themes/palenight-theme.json]: I found!
>  "inactiveColoredFileBackground": "#00000000"

https://plugins.jetbrains.com/docs/intellij/themes-customize.html#customizing-the-borders-of-ui-controls

**Comment by lankydan** [comment]
If you delete all the classes in/under the package `com/github/lankydan/palenightintellijtheme` then the verification errors go away. Those classes were copied from the template when I created the plugin and doesn't look like they are needed.

**Comment by mrakofff** [comment]
@MilanovaSmile How about fix, do you have any problems with finishing?

**Comment by lankydan** [comment]
I'll try get round to it, sorry just been hard to find the time.

**Comment by lankydan** [comment]
A new version is pending approval with Jetbrains, so it should be available soon.

**Pull request merged**